lib/scripts/repo_export_in_batches.rb

```diff
@@ -0,0 +1,44 @@
+# Before you run this script empty the directory tmp/export/!
+#   rm -rf tmp/export/; mkdir tmp/export/
```

Comment:
Do we need to empty the directory? I may be reluctant to knowing how long it has been taking. Takes a while to create the exported json files, and one second to remove them all 😨

Reply:
Good point. I guess we can copy the contents out, then remove it.